.github/workflows/pull_request.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
         image: postgres:14.5
         # Provide the password for postgres
         env:
-          POSTGRES_DB: oipie
```

Comment:
why?
please give a justification.

Reply:
We want a PostgreSQL database for testing purposes. Our test environment is pointing to `oipie_tests` database name instead of `oipie`. That's the only reason

Reply:
why?
we dont need another schema for testing.
we need to test using the same schema.